bindings/go/unicorn/uc.c

```diff
@@ -23,3 +23,7 @@ uc_err uc_reg_write_batch_helper(uc_engine *handle, int *regs, uint64_t *val_in,
     free(val_ref);
     return ret;
 }
+
+void uc_ctl_set_cpu_model_helper(uc_engine *handle, int model) {
+    uc_ctl(handle, UC_CTL_CPU_MODEL, model);
```

Comment:
This is wrong, you have to use `uc_ctl_set_cpu_model` macro.

Reply:
Fixed